Video-3977: disable "reacquire track" workaround for screenshare track (#1565)
twilio/twilio-video.js: CHANGELOG.md

---------

- Fixed a bug where the VideoTracks of Safari Participants with VP8 simulcast enabled sometimes had low frame rates. (VIDEO-6263)

> - Fixed a bug which caused screen share track to get restarted as camera track if it was ended when gaining foreground. (VIDEO-3977)

Reviewer: ```suggestion
- Fixed a bug where the screen share track got restarted as a camera track if it was ended when the application was foreground. (VIDEO-3977)
```
